plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -33,14 +33,14 @@
         _source: ["https://www.google.lv", "https://www.google.co.in/", "https://www.google.ru/", "http://stackoverflow.com/questions", "http://stackoverflow.com/unanswered", "http://stackoverflow.com/tags", "http://r.search.yahoo.com/"]
 	};
 	var eventsMap = {
-		"Login": ["Lost", "Won","[CLY]_star_rating"],
+		"Login": ["Lost", "Won"],
 		"Logout": [],
 		"Lost": ["Won", "Achievement", "Lost"],
 		"Won": ["Lost", "Achievement"],
 		"Achievement": ["Sound", "Shared"],
 		"Sound": ["Lost", "Won"],
 		"Shared": ["Lost", "Won"],
-		"[CLY]_star_rating":["Lost", "Won", "Achievement"]
+        "[CLY]_action":[]
```

Comment:
I think there is no point providing action key here, as it will be called specifically, rather than randomly